src/style.css

```diff
@@ -0,0 +1,132 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.block {
+  display: flex;
+  flex-direction: row;
+  font-family: inherit;
+  font-style: normal;
+
+}
+
+.block__card {
+  margin: 28px;
+  width: 198px;
+  height: 400px;
```

Comment:
Don't hardcode exact height size. Add necessary paddings according to mockup and let content dictate the container size.